submissions/kikinovk/OOP_exercise/index.js

```diff
@@ -0,0 +1,108 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Being {
+  constructor(name, gender, saying) {
+
+     this.name = name;
+     this.gender = gender;
+     this.saying = saying;
+     this.friends = [];
+     this.species = '';
+  };
+  addFriends(arr) {
+     arr.forEach(item => this.friends.push(item));
+  };
+  description() {
+     return `${this.saying} my name is ${this.name}, I am a ${this.species}, I am ${this.gender}, ${this.friends.length > 0 ? `My friends is ${ this.friends.map(item => item.name).join(' and ')}` : 'I have not got friends'},`
```

Comment:
Quite a lengthy string.
How about splitting it in logical parts, listing them in an array and joining the result?